libs/design/src/atoms/form/quantity-field/quantity-field.component.ts

```diff
@@ -55,7 +55,15 @@ export class DaffQuantityFieldComponent implements ControlValueAccessor, DaffFor
    */
   @Input() selectMax = 10;
 
-  focused = false;
+  get focused() {
```

Comment:
Type the getter function return: `get focused(): boolean {`

Reply:
updated